rest/route/project.go

```diff
@@ -1048,3 +1048,59 @@ func (h *projectParametersGetHandler) Run(ctx context.Context) gimlet.Responder
 
 	return gimlet.NewJSONResponse(res)
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// PUT /rest/v2/projects/value/{toReplace}/{replacement}
```

Comment:
can this be rotate_value instead? Also we don't read toReplace or replacement in as variables so you can drop that part from this comment